docs/adr/0045-testing-updates-and-recommendations.md

```diff
@@ -0,0 +1,203 @@
+- Audit and Identify Inaccurate/False Tests
+  - **Good**, because we would find tests that are giving potential false positives (or negatives)
+  - **Good**, because we could remove tests that are no longer valid.
+  - **Bad**, because we would still have tests that are long running, testing multiple functionalities instead of being focused. Functional tests are built step on top of step, and a failure early on in a test may not indicate a failure of the thing _being_ tested.
+  </details>
+
+<details>
+<summary>C: Fixing Testing Dashboard</summary>
+
+Options:
+
+1. This is kind of a no-brainer, we need to fix it. [Bug ticket here][dashboard bug ticket]
+
+However, to expand on this, we should consider a more robust process for review and triage of the [Test Metrics dashboard][test metrics dashboard], baking it into our greater development process. There is an opportunity to push towards improving some metrics, such as lowering skip counts, understanding spikes in execution times, lowering retry rates for functional tests, etc.
+
+The dashboard has an opportunity to push metrics about test health into the limelight.
+
+</details>
+
+<details>
+<summary>D: Update and Expand Central Test Documentation</summary>
+asdf
+
+Options:
+
+1. Revise and Extend Existing Docs **👍 Recommended**
+2. Minimal Edits for Accuracy
+
+#### Details
+
+We have most documentation around testing in the [developer docs][developer docs], however test references and patterns are spread about throughout the docs, and we're missing guidance on how to write good tests. As a result, most patterns of testing come down to personal preference, or who is doing a code review. We try to stick to patterns but it can always vary.
+
+Additionally, documentation around _what_ testing exists, or patterns to follow in CI, or tagging, or for functional tests live spread about and may be difficult to find and it's okay to have them closer to the thing they should be, but then linking back to a source could make them easier to find and identify areas that we're missing.
+
+At a minimum however, the documentation we do have may be out of date, or include reference we have not fully adopted, for example the [Test Strategy][test strategy] document. We should review all test related documents, update references and links, and if there are things yet to be fully adopted we should
+
+#### Pros and Cons of each option
+
+- Revise and Extend Existing Docs
+  - **Good**, because we get a single "home" for testing in FxA
+  - **Good**, because participating in testing, be that unit, integration, or functional, becomes easier when there is guidance on how to add and update tests
+  - **Good**, because we also get a place for best practices and patterns, such as how to identify code-smell when making tests, avoiding too many assertions per test, or
```

Comment:
```suggestion
  - **Good**, because we also get a place for best practices and patterns, such as how to identify code-smell when making tests or avoiding too many assertions per test
```